[2023-03-29 19:20:06,791] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v1.extract_data_task manual__2023-03-29T19:19:52.803586+00:00 [queued]>
[2023-03-29 19:20:06,902] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v1.extract_data_task manual__2023-03-29T19:19:52.803586+00:00 [queued]>
[2023-03-29 19:20:06,907] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 19:20:06,912] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2023-03-29 19:20:06,914] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 19:20:07,015] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): extract_data_task> on 2023-03-29 19:19:52.803586+00:00
[2023-03-29 19:20:07,039] {standard_task_runner.py:52} INFO - Started process 688 to run task
[2023-03-29 19:20:07,055] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'SP_500_DATA_PIPELINE_v1', 'extract_data_task', 'manual__2023-03-29T19:19:52.803586+00:00', '--job-id', '186', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp6cna03p5', '--error-file', '/tmp/tmp7l_hnb9d']
[2023-03-29 19:20:07,065] {standard_task_runner.py:77} INFO - Job 186: Subtask extract_data_task
[2023-03-29 19:20:07,442] {logging_mixin.py:109} INFO - Running <TaskInstance: SP_500_DATA_PIPELINE_v1.extract_data_task manual__2023-03-29T19:19:52.803586+00:00 [running]> on host f6db1c430e11
[2023-03-29 19:20:07,786] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=SP_500_DATA_PIPELINE_v1
AIRFLOW_CTX_TASK_ID=extract_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T19:19:52.803586+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T19:19:52.803586+00:00
[2023-03-29 19:20:13,957] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/pandas_datareader/tiingo.py:234: FutureWarning: In a future version of pandas all arguments of concat except for the argument 'objs' will be keyword-only
  return pd.concat(dfs, self._concat_axis)

[2023-03-29 19:21:23,088] {logging_mixin.py:109} INFO - Error while extracting data for BRK.B: Unable to read URL: https://api.tiingo.com/tiingo/daily/BRK.B/prices?startDate=2018-03-30&endDate=2023-03-29&format=json
Response Text:
b'{"detail":"Error: Ticker \'BRK.B\' not found"}'
[2023-03-29 19:21:40,914] {logging_mixin.py:109} INFO - Error while extracting data for BF.B: Unable to read URL: https://api.tiingo.com/tiingo/daily/BF.B/prices?startDate=2018-03-30&endDate=2023-03-29&format=json
Response Text:
b'{"detail":"Error: Ticker \'BF.B\' not found"}'
[2023-03-29 19:29:07,663] {logging_mixin.py:109} INFO - Failed to retrieve data for the following tickers: ['BRK.B', 'BF.B']
[2023-03-29 19:29:10,884] {logging_mixin.py:109} INFO - ingestion from api completed
[2023-03-29 19:29:54,341] {logging_mixin.py:109} INFO - file has been saved at :sp_500_data.csv
[2023-03-29 19:29:54,344] {logging_mixin.py:109} INFO - sp_500_data.csv
[2023-03-29 19:29:54,409] {python.py:175} INFO - Done. Returned value was: None
[2023-03-29 19:29:54,455] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=SP_500_DATA_PIPELINE_v1, task_id=extract_data_task, execution_date=20230329T191952, start_date=20230329T192006, end_date=20230329T192954
[2023-03-29 19:29:54,602] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-29 19:29:54,725] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
